### ISSUE TITLE
doesn't work with geojson schema

### ISSUE THREAD

**Issue by paleozogt** [CDThomas/graphql-json-to-sdl#16]
```
✗ git clone https://github.com/geojson/schema.git geojson-schema
✗ cd geojson-schema
✗ make

...

✗ cd build

✗ ls
ls
FeatureCollection.json  Feature.json  GeoJSON.json  GeometryCollection.json  Geometry.json  LineString.json  MultiLineString.json  MultiPoint.json  MultiPolygon.json  Point.json  Polygon.json

✗ graphql-json-to-sdl GeoJSON.json GeoJSON.graphql
    TypeError: Cannot read property '__schema' of undefined
```

**Comment by Wrent** [comment]
I had the same issue, but wrapping the entire `__schema` in `data` made it work.

So I had this json:
```json
{
  "__schema": {
    ...
  }
}
```
and I changed it to this:
```json
{
  "data": {
    "__schema": {
      ...
    }
  }
}
```

**Comment by CDThomas** [comment]
Hi @paleozogt, `graphql-json-to-sdl` is intended to take a JSON GraphQL schema from a GraphQL introspection query result and output the corresponding SDL schema.

I'm not familiar with GeoJSON Schema, but I took a look at https://github.com/geojson/schema and it looks like it uses a different kind of JSON schema (described by http://json-schema.org/). `graphql-json-to-sdl` does not support http://json-schema.org/ schemas.

I can see how the name might cause some confusion and the error message certainly could be improved.

**Comment by CDThomas** [comment]
> I had the same issue, but wrapping the entire `__schema` in `data` made it work.
> 
> So I had this json:
> 
> ```json
> {
>   "__schema": {
>     ...
>   }
> }
> ```
> 
> and I changed it to this:
> 
> ```json
> {
>   "data": {
>     "__schema": {
>       ...
>     }
>   }
> }
> ```

@Wrent did you run into this issue using https://github.com/geojson/schema or was this from another project? Was the JSON you were using for input from a GraphQL introspection query?

**Comment by Wrent** [comment]
> @Wrent did you run into this issue using https://github.com/geojson/schema or was this from another project? Was the JSON you were using for input from a GraphQL introspection query?

No, it was from a custom project and the JSON was generated from introspection query: `yarn run apollo schema:download --endpoint=https://localhost:61000/graphql --skipSSLValidation graphql-schema.graphql`

The GraphQL backend is backed by `graphql-java`

**Comment by CDThomas** [comment]
Closing this issue since the issue with GeoJSON schema looks unrelated to the missing `data` property error for a legitimate   GraphQL schema.

It looks like https://github.com/CDThomas/graphql-json-to-sdl/pull/41 would fix the error you've described @Wrent, so feel free to leave a comment there or open another issue.
